cypress/e2e/project-list.cy.ts

```diff
@@ -20,19 +20,45 @@ describe("Project List", () => {
       cy.viewport(1025, 900);
     });
 
-    it("renders the projects", () => {
+    it("renders the projects and has correct statues", () => {
       const languageNames = ["React", "Node.js", "Python"];
 
       // get all project cards
       cy.get("main")
         .find("li")
         .each(($el, index) => {
           // check that project data is rendered
+          let expectedColor;
+          let expectedText;
+          let badgeColor;
+
+          switch (mockProjects[index].status) {
+            case "error":
+              expectedColor = "rgb(254, 243, 242)";
+              expectedText = "critical";
+              badgeColor = "error";
+              break;
+            case "info":
+              expectedColor = "rgb(236, 253, 243)";
+              expectedText = "stable";
+              badgeColor = "success";
+              break;
+            case "warning":
+              expectedColor = "rgb(255, 250, 235)";
+              expectedText = "warning";
+              badgeColor = "warning";
+              break;
+          }
+
           cy.wrap($el).contains(mockProjects[index].name);
           cy.wrap($el).contains(languageNames[index]);
           cy.wrap($el).contains(mockProjects[index].numIssues);
           cy.wrap($el).contains(mockProjects[index].numEvents24h);
-          cy.wrap($el).contains(capitalize(mockProjects[index].status));
+          cy.wrap($el)
+            .find(`[data-testid="badge-${badgeColor}"]`)
+            .should("exist")
+            .and("have.css", "background-color", expectedColor)
+            .and("contain", capitalize(expectedText));
```

Comment:
You could remove the `capitalize` function here and adjust the `expectedText` above accordingly.